packages/election/lib/elections.js

```diff
@@ -0,0 +1,67 @@
+const debug = require('debug')('elections:lib:elections')
+
+const getConditions = () => ({
+  active: true,
+  'endDate >': Date.now()
+})
```

Comment:
why should past elections not be visible on the api anymore? how to access the results?